content/en/docs/tasks/traffic-management/ingress/ingress-sni-passthrough/index.md

```diff
@@ -304,10 +304,15 @@ to set the `SECURE_INGRESS_PORT` and `INGRESS_HOST` environment variables.
 
 {{< tab name="Gateway API" category-value="gateway-api" >}}
 
+Wait for the Gateway to be accepted:
+
+{{< text syntax=bash snip_id=none >}}
+$ kubectl wait --for=condition=accepted gtw mygateway
```

Comment:
The correct semantics of gateway accepted vs programmed vs ready is still not agreed in the upstream gateway-api project, so I think we should wait for resolution before updating any docs.
https://github.com/kubernetes-sigs/gateway-api/issues/1877

Reply:
@frankbu PTAL at https://github.com/istio/istio.io/pull/12955#issuecomment-1485711217. The way that Gateway status is currently implemented `Programmed=True` can be surfaced when an address is unassigned. This is causing tests that set `INGRESS_HOST` to fail. As you mention, kubernetes-sigs/gateway-api#1877 is attempting to resolve ambiguity in the conditions. Maybe the best path forward is to check for the presence of both status conditions until this is resolved?

Reply:
@danehans The problem is that there are 14 other docs that are broken by the current implementation. Fixing the implementation (to at least align with the best guess of where the spec is going) seems like a better approach than hacking all the docs temporarily. (Btw, the fact that the current impl has accepted not true until the address is assigned is an obvious bug since according to the spec accepted just means the config is syntactically valid.)